pkg/dockerutil/dockerutils.go

```diff
@@ -45,6 +46,19 @@ import (
 // NetName provides the default network name for ddev.
 const NetName = "ddev_default"
 
+// DockerVersionConstraint is the current minimum version of Docker recommended for DDEV.
+// If this constraint is not met, we will warn the user, but not fail.
```

Comment:
This isn't true now is it?

```suggestion
```

Reply:
It's a warning, not an error:

https://github.com/ddev/ddev/blob/e6bf644d6c93216a4fa754b527b169fcd95920fa/cmd/ddev/cmd/root.go#L64-L73

We need to either remove this check from `root.go` or leave the comment as is.